react/components/EXPERIMENTAL_Conditions/Statement.js

```diff
@@ -58,6 +58,15 @@ class Statement extends React.Component {
     this.handleChangeStatement(null, 'error')
   }
 
+  handleKeyPress = (event, handler) => {
+    const SPACE = ' '
+    const ENTER = 'Enter'
```

Comment:
Are these really needed? 🤔

Reply:
yes, because if not when the component is in focus, any key that is pressed... will trigger this event.